app/forms/assessor_interface/reference_request_form.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+class AssessorInterface::ReferenceRequestForm
```

Comment:
Not a big deal, and we can always do it later, but could we make this form generic? If it accepted a `reviewable` rather than a `reference_request` we could apply it to the requestables we'll be reviewing later on.